pkg/sql/executor.go

```diff
@@ -759,9 +759,7 @@ func (e *Executor) execParsed(
 		// that is encountered resets stmts.
 
 		inTxn := txnState.State() != NoTxn
-		execOpt := client.TxnExecOptions{
-			AssignTimestampImmediately: true,
```

Comment:
did you forget to actually remove `AssignTimestampImmediately` from the struct, though?